clayui.com/content/docs/get-started/using-clay-in-jsps.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Using Clay in JSPs'
+order: 6
+---
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [Setting up your module](#setting-up-your-module)
+-   [Using Clay taglib in your JSPs](#using-clay-taglib-in-your-jsps)
+-   [Clay components available in JSPs](#clay-components-available-in-jsps)
+
+</div>
+</div>
+
+Clay in combination with DXP provides a set of tags for creating Clay components in your JSP files.
```

Comment:
I would like to make things more explicit here that the version contained in the JSP are v2 APIs and that it uses v2 components and it would be great to make it clear that the JSPs are only available in Liferay DXP, a link to the product would be great too.

Reply:
Just to know, I didn't follow the taglib changes, are those listed using the components in v3?

Reply:
The ones listed follow the v3 APIs.

Just to clarify, many of these tags DO NOT USE the React Components. They simply render basic markup and mimic the React API since we don't have a stable and performant way of rendering them from Java.

Reply:
Thanks @jbalsas for clarifying.